src/Intercom/Data/Visitor.cs

```diff
@@ -13,30 +13,8 @@
 
 namespace Intercom.Data
 {
-    public class Visitor : Model
+    public class Visitor : User
     {
-        public string user_id { get; set; }
```

Comment:
I am asking alot fo stupid questions here so apologies but why are you changing the class?

Reply:
The current Model class only has:

```csharp
public virtual string id { set; get; }
public virtual string type { get; set; }
```

While we need more than that, as the visitors endpoint now returns more data. By inheriting from the User now we get access to all these here: [Visitor Model](https://developers.intercom.com/v2.0/reference#visitor-object). This avoids duplicating code.